Mathlib/Algebra/AddConstMap/Basic.lean

```diff
@@ -118,15 +118,15 @@ theorem map_nat' [AddMonoidWithOne G] [AddMonoid H] [AddConstMapClass F G H 1 b]
 
 theorem map_ofNat' [AddMonoidWithOne G] [AddMonoid H] [AddConstMapClass F G H 1 b]
     (f : F) (n : ℕ) [n.AtLeastTwo] :
-    f (OfNat.ofNat n) = f 0 + (OfNat.ofNat n : ℕ) • b :=
+    f (ofNat n) = f 0 + (ofNat(n) : ℕ) • b :=
   map_nat' f n
 
 theorem map_nat [AddMonoidWithOne G] [AddMonoidWithOne H] [AddConstMapClass F G H 1 1]
     (f : F) (n : ℕ) : f n = f 0 + n := by simp
 
 theorem map_ofNat [AddMonoidWithOne G] [AddMonoidWithOne H] [AddConstMapClass F G H 1 1]
     (f : F) (n : ℕ) [n.AtLeastTwo] :
-    f (OfNat.ofNat n) = f 0 + OfNat.ofNat n := map_nat f n
+    f (ofNat n) = f 0 + ofNat n := map_nat f n
```

Comment:
```suggestion
    f ofNat(n) = f 0 + ofNat(n) := map_nat f n
```
etc. The bare `ofNat` is either `Int.ofNat` or `OfNat.ofNat`, neither of which being what you want.